modules/wsl-distro.nix

```diff
@@ -153,7 +156,7 @@ with lib; {
         })
         (mkIf cfg.nativeSystemd {
           wsl.wslConf = {
-            user.default = cfg.defaultUser;
+            user.default = config.users.users.${cfg.defaultUser}.name;
```

Comment:
ah, probably because my user.users attr matches name.

Reply:
Changing the username has always worked fine, but having different username and attr hasn't